src/controllers/connectionManager.ts

```diff
@@ -245,12 +245,19 @@ export default class ConnectionManager {
     }
 
     /**
-     * Get the connection view.
+     * Exposed for testing purposes
```

Comment:
Also wasn't the getter already exposed? Maybe revert back the comment for the getter.